src/site/content/en/progressive-web-apps/drive-business-success/index.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: post
+title: How Progressive Web Apps can drive business success
+authors:
+  - sfourault
+date: 2020-05-20
+updated: 2020-05-20
+description: Build a solid business case for your PWA. Learn when you should invest, and how you can measure it's success.
+---
+
+Progressive Web Apps are on a lot of companies' roadmap to modernize their
+website and adapt to users' new expectations. Like all new concepts and
+technical capabilities, they raise questions: is it what my customers want,
+how much will it grow my business, what is technically feasible?
+
+![Identify your stakeholders](img-1.jpg)
+
+To shape your digital strategy, several stakeholders are often involved: the
+product manager and CMO are co-owners of the business impact of each feature,
```

Comment:
```suggestion
Product Manager and CMO are co-owners of the business impact of each feature,
```
Just keeping capitalization consistent since it's capitalized in the image and "UX Researcher" is capitalized below